src/SFA.DAS.EmploymentCheck.Application.UnitTests/Services/EmploymentCheckServiceTests/WhenStoreCompletedCheck.cs

```diff
@@ -70,11 +70,15 @@ public async Task Then_UpdateEmploymentCheckAsComplete_is_called()
             var request = _fixture.Create<EmploymentCheckCacheRequest>();
             var response = _fixture.Create<EmploymentCheckCacheResponse>();
 
+            _employmentCheckRepositoryMock
+                .Setup(x => x.UpdateEmploymentCheckAsComplete(It.IsAny<Data.Models.EmploymentCheck>(), _unitOfWorkMock.Object))
+                .Returns(Task.CompletedTask);
+
             // Act
             await _sut.StoreCompletedCheck(request, response);
 
             // Assert
-            _employmentCheckRepositoryMock.Verify(x => x.UpdateEmploymentCheckAsComplete(request, _unitOfWorkMock.Object), Times.Once());
+            _employmentCheckRepositoryMock.Verify(x => x.UpdateEmploymentCheckAsComplete(It.IsAny<Data.Models.EmploymentCheck>(), _unitOfWorkMock.Object), Times.Once());
```

Comment:
not a good use of It.IsAny<Data.Models.EmploymentCheck>() if you want to verify correct data is being sent. instead, use ` It.Is<Data.Models.EmploymentCheck>(check => check..... == ... & ... ===..._)`

Reply:
Updated to use It.Is<Data.Models.EmploymentCheck>(check => check..... == ... & ... ===..._)